app/uk/gov/hmrc/transitmovements/services/MessageFactory.scala

```diff
@@ -80,6 +86,22 @@ class MessageFactoryImpl @Inject() (
         )
     }
 
+  def createEmptyMessage(
+    messageType: MessageType,
+    received: OffsetDateTime,
+    status: MessageStatus
+  ): Message =
+    Message(
+      id = MessageId(ShortUUID.next(clock, random)),
+      received = received,
+      generated = None,
+      messageType = messageType,
+      triggerId = None,
+      url = None,
+      body = None,
+      status = status
```

Comment:
status here will always be pending, so rather than passing it as a parameter, just set it in the object creation

`status = Pending`